about-adas-html-css-lessons/welcome.md

```diff
@@ -0,0 +1,11 @@
+# Welcome!
+
+In the transition between Ada Developer Academy's textbook curriculum to core curriculum, we lost a lot, and we preserved a lot.
+
+The HTML and CSS curriculum has not changed significantly.
+
+The order of the material and lessons has changed a lot. The old material has been broken into a different flow.
+
+However, no new content has changed, except for the addition of the Problem Set in "Structuring Content."
+
+If you're looking for another way to view our HTML/CSS material, consider [this record of Ada videos from C14 (Fall 2020)](https://adaacademy.hosted.panopto.com/Panopto/Pages/Sessions/List.aspx#folderID=%2214581e7b-e913-4a4d-b1d7-ac4b01449065%22).
```

Comment:
How is this expected to be included? It sounds pretty melancholic, and I'm not sure a current reader would even be aware of the textbook curriculum.

I offer my own take that still plays around with the idea of change (constants and variables) but I think is a little less dismissive of the work that's gone into this ongoing work. (and also deals with "no new content has changed")

```suggestion
Ada Developers Academy's curriculum has seen many changes from cohort to cohort.

Our HTML and CSS curriculum has remained as a relative constant in a sea of variability.

But the only constant in life is change, and even for our HTML and CSS curriculum, change was inevitable. We have revisited it and determined that it could benefit from a change in the order of presentation.

Beyond the order, the most significant change is in the inclusion of a Problem Set for "Structuring Content."

So for another perspective on our HTML/CSS material, consider [this record of Ada videos from C14 (Fall 2020)](https://adaacademy.hosted.panopto.com/Panopto/Pages/Sessions/List.aspx#folderID=%2214581e7b-e913-4a4d-b1d7-ac4b01449065%22).
```